package.json

```diff
@@ -1,12 +1,16 @@
 {
-  "name": "razeedash-api",
-  "version": "0.6.3",
+  "name": "@razee/razeedash-api",
+  "version": "0.0.0-0",
```

Comment:
why is this set to `0.0.0-0` ?

Reply:
Denotes that the version doesn't come from github. Its supplied when we tag a build for release.